### ISSUE TITLE
Unity 2021.3.2f1 doesn't work

### ISSUE THREAD

**Issue by p0we7** [kiraio-moe/remove-unity-splash-screen#1]
Hi, kiraio
I tried to remove the unity splash screen based on your GitHub repo, but it didn't work. My unity version is 2021.3.2f1, and I found the first binary 01 via hex, but when I open globalgamemanagers with UABE, the UABE can't parse the file.

I have uploaded the corresponding file in the attachment, hope you can study it if you can, thanks.

[globalgamemanagers.zip](https://github.com/kiraio-moe/remove-unity-splash-screen/files/10471711/globalgamemanagers.zip)

**Comment by kiraio-moe** [comment]
Hi, thanks for reporting.

I have checked the file and yes, UABE didn't support 2021^.
Also, the splash screen is still showing because you haven't changed the ``hasPROVersion`` boolean to ``true``.

I'll add the tutorial soon as possible how to change it through the Hex editor.

**Comment by kiraio-moe** [comment]
So, I have tried to change the ``m_ShowUnitySplashScreen`` and ``hasPROVersion`` boolean, give it a try.
[globalgamemanagers-mod.zip](https://github.com/kiraio-moe/remove-unity-splash-screen/files/10472728/globalgamemanagers-mod.zip)

Boolean ``hasPROVersion`` is likely in offset ``AEBC`` (14 offset before the highlighted Unity version). I have changed it to ``01`` in the attachment:

![pro_version](https://user-images.githubusercontent.com/58289710/213880649-d3bf430e-55ad-4805-878d-5fbc6b03ee7f.png)

If it's not work, then the boolean is somewhere between the SampleScene and Unity version. Try swapping the ``00`` value to ``01``.
